Adds the User appointment scheduling form

aayush-musyaju opened the pull request:
The registered user can use the form to schedule an appointment for
vaccination

aayush-musyaju committed:
Adds the User appointment scheduling form

The registered user can use the form to schedule an appointment for
vaccination

aayush-musyaju committed:
Adds the user appointments list page

Users can now see all of their appointments from the appointments list page.

aman007shrestha reviewed (approved):
well written. Looks good to me!

aayush-musyaju committed:
Makes the vaccination list is appointment form dynamic

All the vaccinations list are stored in an array which is then mapped
and converted into the Options.
